portfolio/src/main/webapp/index.html

```diff
@@ -162,6 +161,24 @@ <h2>Latest Reports</h2>
             <input id="searchBox-input" type="text" placeholder="Enter a location"/>
         </div>
     </div>
+
+    <div class="search-route" id="search-route">
+        <div id="b">Directions API</div>
+            <select id "travel" size = "3">
```

Comment:
missing `=` between `id` and `travel`
I don't think you need an id here since nothing could be using it